projects/smart-table/src/lib/smart-table/smart-table.component.ts

```diff
@@ -23,41 +21,51 @@ import {
   SmartTableOptions,
   UpdateFilterArgs,
 } from './smart-table.types';
+import {first, map, tap} from 'rxjs/operators';
+import {PROVIDE_ID} from '../indentifier.provider';
+import {Observable, of} from 'rxjs';
 
 @Component({
   selector: 'aui-smart-table',
   styleUrls: ['./smart-table.component.scss'],
   templateUrl: './smart-table.component.html'
 })
-export class SmartTableComponent implements AfterViewInit {
+export class SmartTableComponent {
   @Input() apiUrl: string;
   @Input() httpHeaders: HttpHeaders;
   @Input() columnTypes: SmartTableColumnCustomType[] = [];
+
   @Input()
   set configuration(configuration: SmartTableConfig) {
```

Comment:
ik begrijp niet goed wanneer of waarom dit opgeroepen wordt als je geen configuration input meegeeft?